airflow/providers/celery/executors/celery_executor.py

```diff
@@ -267,6 +267,7 @@ def __init__(self):
         self.tasks = {}
         self.task_publish_retries: Counter[TaskInstanceKey] = Counter()
         self.task_publish_max_retries = conf.getint("celery", "task_publish_max_retries")
+        self.send_pool = ProcessPoolExecutor(max_workers=self._sync_parallelism)
```

Comment:
Don't we need to gracefully shutdown it?

Reply:
When I did performance testing, I also used a dag composed of a simple BashOperator. You may not see the difference when the load is relatively low. When I tested, the load of the airflow scheduler was very high. There were 40,000 scheduled dags (every 2 hours) and 10,000 manually triggered dags. The number of schedulers was 30, and the number of workers was 100.